internal/configs/configurator.go

```diff
@@ -1484,6 +1518,95 @@ func (cnf *Configurator) RefreshAppProtectUserSigs(
 	return allWarnings, cnf.reload(nginx.ReloadForOtherUpdate)
 }
 
+func appProtectDosPolicyFileNameFromUnstruct(unst *unstructured.Unstructured) string {
+	return fmt.Sprintf("%s%s_%s.json", appProtectDosPolicyFolder, unst.GetNamespace(), unst.GetName())
+}
+
+func appProtectDosLogConfFileNameFromUnstruct(unst *unstructured.Unstructured) string {
+	return fmt.Sprintf("%s%s_%s.json", appProtectDosLogConfFolder, unst.GetNamespace(), unst.GetName())
+}
+
+// DeleteAppProtectDosPolicy updates Ingresses and VirtualServers that use AP Dos Policy after that policy is deleted
+func (cnf *Configurator) DeleteAppProtectDosPolicy(polNamespaceName string, ingExes []*IngressEx, mergeableIngresses []*MergeableIngresses, vsExes []*VirtualServerEx) (Warnings, error) {
+	if len(ingExes)+len(mergeableIngresses)+len(vsExes) > 0 {
```

Comment:
Would it be a good idea to create a function that generates the name of the file? We can use this new function in both places we need the file name, where the file is created and where it is deleted. In this way, they will always match?

Reply:
Done both